Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -127,7 +129,26 @@ internal static Result CreateFileResult(string filePath, Query query, int score
                 {
                     try
                     {
-                        if (c.SpecialKeyState.CtrlPressed)
+                        if (File.Exists(filePath) && c.SpecialKeyState.CtrlPressed && c.SpecialKeyState.ShiftPressed)
+                        {
+                            Task.Run(() =>
```

Comment:
Is adding `Task.Run` help responsibility?

Reply:
yes it does, I tested this using a batch file that wait for 10 seconds before exiting. Without Task.Run the UI freezes for 10 seconds until the batch file is done executing then it hides itself. With Task.Run the UI hides itself immediately and is ready to be used again while the batch file is still running.